test/EndToEnd/nuget.assert.ps1

```diff
@@ -721,13 +722,21 @@ function Assert-NetCorePackageReference {
         [parameter(Mandatory = $true)]
         [string]$Id,
         [parameter(Mandatory = $true)]
-        [string]$Version        
+        [string]$Version,
+        [bool]$VersionIsElement        
     )
 
     $doc = [xml](Get-Content $project.FullName)
-    $references = $doc.SelectNodes("./Project/ItemGroup/PackageReference[@Include = '$id' and @Version = '$Version']")
-    
-    Assert-True ($references.Count -eq 1) "Project $($project.FullName) does not contain a reference to Package $($Id) $($Version)"
+    if($VersionIsElement)
```

Comment:
That's why it's important. Person writing a test shouldn't care about particular details. Also this will likely change in future.